src/[sandbox]/sandbox.ts

```diff
@@ -81,7 +82,7 @@ export class Sandbox extends LitElement {
             <h2>cjaas-timeline</h2>
             ${timelineTemplate}
           </div>
-          <div class="container" aria-label="cjaas-profile">
+          <!-- <div class="container" aria-label="cjaas-profile">
```

Comment:
Maybe clean up the commented out code.

Reply:
Elix list explorer does not work with firefox for some reason. Only the first widget shows up. So I had other widgets disabled.